packages/legend-graph/src/graphManager/protocol/pure/v1/engine/query/V1_QuerySearchSpecification.ts

```diff
@@ -42,6 +42,7 @@ export class V1_QuerySearchSpecification {
   stereotypes?: V1_StereotypePtr[] | undefined;
   limit?: number | undefined;
   showCurrentUserQueriesOnly?: boolean | undefined;
+  showLatestQueriesFirst?: boolean | undefined;
```

Comment:
we are getting rid of this flag in engine right?

Reply:
yes